packages/block-editor/src/components/convert-to-group-buttons/toolbar.js

```diff
@@ -32,16 +32,31 @@ function BlockGroupToolbar( { label = _x( 'Group', 'verb' ) } ) {
 		[ clientIds ]
 	);
 
-	const onConvertToGroup = () => {
+	const variationAttributes = {
+		row: { type: 'flex' },
+		stack: { type: 'flex', orientation: 'vertical' },
+	};
```

Comment:
suggestion (non-blocking): Maybe move this outside of the component. No need to recreate it every time it re-renders.

Reply:
ah, true, thanks, done.